backend_new/.env.template

```diff
@@ -34,4 +34,6 @@ TWILIO_AUTH_TOKEN=
 PARTNERS_PORTAL_URL=http://localhost:3001/
 # sendgrid email api key
 EMAIL_API_KEY=SG.ExampleApiKey
+# controls the repititon of the afs cron job
+AFS_PROCESSING_CRON_STRING=0,3,15 * * * *
```

Comment:
question: for clarification this means it will run at minute 0, 3 and 15, correct?

suggestion: if my understanding is correct, recommend having cron run at more even intervals like "0, 15, 30, 45" or something like "15, 45", or just once an hour. Not sure what the optimal frequency will be that won't cause performance issues under heavy load.

Reply:
That's a really good call out, on production we use `0 0 * * *` as the value 
which we mean every day at midnight server time this would execute 
we add the seconds as a random number 

Right now we have this run so frequently locally so that we can catch any issues with the process job, I can change the suggestion in the .env.template though if that would make things more clear

Reply:
ah, thanks for the clarification. I think it's ok to keep what's in the template, but maybe add to the comment what is recommended for production environments.